lib/admin.php

```diff
@@ -338,5 +338,73 @@ function woocommerce_embed_page_header() {
 	</div>
 	<?php
 }
-
 add_action( 'in_admin_header', 'woocommerce_embed_page_header' );
+
+/**
+ * Registers WooCommerce specific user data to the Wordpress user API.
```

Comment:
[capital_P_dangit](https://codex.wordpress.org/Function_Reference/capital_P_dangit)() 😄

Reply:
Haha, I'm guilty of copypasta here, honest!

Reply:
Fixed in 8ba7438